style-guides.adoc

```diff
@@ -0,0 +1,18 @@
+= Boost Style Guide
+
+This document outlines the style guidelines for the site-docs. Adhering to these
+guidelines will help ensure consistency and cohesion across all pages of the website.
+
+
+== UI Controls
+[disc]
+* Use a clear and legible font for text.
+* Keep text within buttons and other UI elements concise and descriptive.
+* Use a contrasting color for buttons to make them stand out, but avoid using the color mustard.
+* Use borders and shadows to create depth and distinguish elements.
+* Ensure buttons and other interactive elements are easily clickable and tappable on mobile devices.
+
+== Conclusion
+
+By following these guidelines, we can maintain a consistent and recognizable brand identity,
+while also ensuring a positive user experience for all visitors to the site.
```

Comment:
These guidelines are generic they shouldn't be here except for the note about using boost mustard.